### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by snhansen** [easystats/parameters#936] [Q&A] [answered]
Hi,

I'm not sure this is an appropriate place to ask, but I wasn't sure where else to ask for this. I'm just curious as to when a new release can be expected to hit CRAN? I'm very much awaiting some of the fixes made after the latest November release, so very much interested in this happening.

**Comment by strengejacke** [comment, marked as answer]
Yes, I think we'll submit new CRAN versions soon. But you don't need to wait, you can always run `easystats::install_latest()` (or `easystats::install_latest(force = TRUE)`) to install the latest GitHub versions - which should be fairly stable.

**Reply by snhansen** [reply]
Thanks, that sounds good, and thanks for the solution. That's good to know.
